skimage/transform/pyramids.py

```diff
@@ -142,7 +133,7 @@ def pyramid_expand(image, upscale=2, sigma=None, order=1,
 
 
 def pyramid_gaussian(image, max_layer=-1, downscale=2, sigma=None, order=1,
-                     mode='reflect', cval=0, multichannel=None):
+                     mode='reflect', cval=0, multichannel=False):
```

Comment:
I think you need to remove the call to the respective multichannel function below.